### PR TITLE
Updated JiraV2 integration to combine issueJson with the transition a…

### PR DESCRIPTION
**Pull request by MWEJ**
…nd status, instead of doing it before. This makes it possible to fill out the resolution field while doing a transition to closed status

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Updated integration to combine issueJson with the transition and status, instead of doing it before. This makes it possible to fill out the resolution field while doing a transition to closed status

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [X ] Yes
       - Further details:
   - [ ] No

## Must have
- [X ] Tests
- [X ] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV2/JiraV2.py [hunk 1]
```diff
@@ -728,29 +728,31 @@ def create_issue_command():
...
+        url = f'rest/api/latest/issue/{issue_id}/'
+        jira_req('PUT', url, json.dumps(issue))
     return get_issue(issue_id, headers, is_update=True)
 
 
-def edit_status(issue_id, status):
+def edit_status(issue_id, status, issue):
     # check for all authorized transitions available for this user
     # if the requested transition is available, execute it.
+    if issue is None:
```
Comment (Noy-Maimon) [inline Packs/Jira/Integrations/JiraV2/JiraV2.py:747]:
```suggestion
    if not issue:
```

File: Packs/Jira/ReleaseNotes/2_0_5.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Updated integration to combine issueJson with the transition and status, instead of doing it before. This makes it possible to fill out the resolution field while doing a transition to closed status
```
Comment (Noy-Maimon) [inline Packs/Jira/ReleaseNotes/2_0_5.md:4]:
```suggestion
- You can now fill modify the resolution field while transitioning a status to closed.
```

**Review by Noy-Maimon — CHANGES_REQUESTED** [review]
Hi, thank you for your contribution - looks great. please note my comments, and please sign the [contribution form](https://forms.gle/XDfxU4E61ZwEESSMA).
Also, let's schedule a demo for this change.

**Comment by Noy-Maimon** [comment]
@MWEJ Hi, any updates?

**Comment by MWEJ** [comment]
> @MWEJ Hi, any updates?

Sorry, I wasn't aware that I had to accept the suggestions. I believe I already signed the contributor form, so is there anything else I need to do?

**Comment by Noy-Maimon** [comment]
@MWEJ Thanks for signing the form, can you please merge for master? this is holding the builds from passing
thanks!

**Review by Noy-Maimon — APPROVED** [review]

**Pull request merged**